src/omv/ports/stm32/stm32fxxx_hal_msp.c

```diff
@@ -188,12 +188,12 @@ void HAL_MspInit(void)
     OMV_AXI_QOS_LTDC_W_SET(OMV_AXI_QOS_LTDC_W_PRI);
     #endif
 
-    #if defined(DCMI_RESET_PIN) || defined(DCMI_PWDN_PIN) || defined(DCMI_FSYNC_PIN)
```

Comment:
This makes sure that if none of the pins are defined, the unused struct doesn't cause a build error

Reply:
Ah, yes, true.

Reply:
Maybe best to make a struct per pin, ex: gpio_reset_init, gpio_fsync_init etc...